### PR TITLE
fix: removed mkdir keystore before rln registration run

### PR DESCRIPTION
**Pull request by NagyZoltanPeter**
It was found on sandbox machine that creating the keystore dir ahead of docker run can cause permission issue writing in it. 
Docker run will create the missing directory by its own when mounting it.

I made several tries.
On sandbox machine it is only working when no keystore directory exists before docker run of wakunode.
On my local machine it works both ways.

### CODE REVIEW

**Review by gabrielmer — APPROVED** [review]
That's amazing! Thanks so much!

@Ivansete-status maybe this was the cause of all the permission issues we saw?

File: register_rln.sh [hunk 1]
```diff
@@ -19,7 +17,7 @@ fi
 
 
 
-docker run -v $(pwd)/keystore:/keystore/:Z wakuorg/nwaku:v0.21.3 generateRlnKeystore \
+docker run -v $(pwd)/keystore:/keystore/:Z wakuorg/nwaku:v0.24.0 generateRlnKeystore \
```
Comment (Ivansete-status) [inline register_rln.sh:20]: Shall we put this in a separate PR?

**Comment by Ivansete-status** [comment]
> That's amazing! Thanks so much!
> 
> @Ivansete-status maybe this was the cause of all the permission issues we saw?

This issue has helped discover the possible reason for the one you mentioned, Gabriel (https://github.com/waku-org/nwaku-compose/issues/32.)

We are waiting for @NagyZoltanPeter to be added to the `adm` group in the sandbox machine, and, if after that he can register properly even having the `mkdir -p keystore` command, that would mean that we are closer to understand the problem of "Cannot open file for writing".

**Comment by NagyZoltanPeter** [comment]
> > That's amazing! Thanks so much!
> > @Ivansete-status maybe this was the cause of all the permission issues we saw?
> 
> This issue has helped discover the possible reason for the one you mentioned, Gabriel (#32.)
> 
> We are waiting for @NagyZoltanPeter to be added to the `adm` group in the sandbox machine, and, if after that he can register properly even having the `mkdir -p keystore` command, that would mean that we are closer to understand the problem of "Cannot open file for writing".

Actually I did try it before issued this PR. Even with adm group involvement it just runs the same for me.
Meaning creating directory from the script (with user creds) before docker run ends in write permission error.
Deleting the directory first and let docker create the keystore dir runs OK.
As having this case common in local machine and in sandbox better remove that mkdir.

File: register_rln.sh (hunk 1 above)
Comment (NagyZoltanPeter) [reply register_rln.sh:20]: Yeah I can... no matter.
But I do agree to create a list where we reference some release version throughout of our repos. Updating shall be part of release train.

Comment (NagyZoltanPeter) [reply register_rln.sh]: Reverted here!

**Review by Ivansete-status — APPROVED** [review]
LGTM! Thanks!
